scripts/prepare-package.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('fs-extra');
+const compat = require('../utils/node-js/compat');
```

Comment:
This would similarly be used by other modules, to accomplish the same "compat" behavior:

```
const compat = require('@skyux/core/utils/node-js/compat');
compat.injectLocaleResourcesContents();
```